crates/std_detect/src/detect/os/linux/riscv.rs

```diff
@@ -0,0 +1,162 @@
+//! Run-time feature detection for RISC-V on Linux.
+
+use super::auxvec;
+use crate::detect::{bit, cache, Feature};
+
+/// Try to read the features from the auxiliary vector, and if that fails, try
+/// to read them from /proc/cpuinfo.
```

Comment:
Reading from the auxiliary vector should never fail, except on some old Android versions which don't support RISC-V anyways. There's no need to parse the contents of `/proc/cpuinfo`.

Reply:
Thanks for providing information! I'd remove /proc/cpuinfo detection for that reasom.